pkg/reconciler/openshift/tektonresult/extension.go

```diff
@@ -18,24 +18,47 @@ package tektonresult
 
 import (
 	"context"
+	"os"
+	"path/filepath"
 
 	mf "github.com/manifestival/manifestival"
 	"github.com/tektoncd/operator/pkg/apis/operator/v1alpha1"
-	"github.com/tektoncd/operator/pkg/client/clientset/versioned"
 	operatorclient "github.com/tektoncd/operator/pkg/client/injection/client"
 	"github.com/tektoncd/operator/pkg/reconciler/common"
+	"github.com/tektoncd/operator/pkg/reconciler/kubernetes/tektoninstallerset/client"
 	occommon "github.com/tektoncd/operator/pkg/reconciler/openshift/common"
+	"knative.dev/pkg/logging"
+)
+
+const (
+	// manifests console plugin yaml directory location
+	routeRBACYamlDirectory = "static/tekton-results/route-rbac"
```

Comment:
why are we adding this `static` prefix? I saw at one more place may be in console PR cc @jkandasa

Reply:
@piyush-garg While writing console plugin PR(#1895), Initially I placed the console plugin files under `cmd/openshift/operator/kodata/tekton-results` and I noticed this directory will be removed by https://github.com/tektoncd/operator/blob/main/hack/fetch-releases.sh#L77, hence I created the manifest files under `cmd/openshift/operator/kodata/static/tekton-results`, but laer I moved the console plugin logic under `tekton-config`.

So now, I have asked @khrm to move the manifest file under `cmd/openshift/operator/kodata/static/tekton-results`, to avoid the deletion on `release.yaml` updates.

When creating the `static` directory, I was thinking to move all the pre-defined(except component release manifest directories) under the `static` directory. That was my original plan.
here `static` means - under this directory files will not be updated by `fetch-releases.sh` script and those are all managed by us.